frontend-cert/js-projects/lab-madlibs/user-stories.md

```diff
@@ -0,0 +1,15 @@
+1. You should define the following variables using `let`:
+- `adjective`
+- `noun`
+- `verb`
+- `place`
+- `adjective2`
+- `noun2`
+
+1. You should assign the above variables some string values of your choice.
+
+1. You should use the following story template to create a story:
+
+`"Once upon a time, there was a(n) [adjective] [noun] who loved to eat [noun2]. The [noun] lived in a [place] and had [adjective2] nostrils that blew fire when it was [verb]."`;
+
+1. You should print the final story to the console.
```

Comment:
There was an issue opened in the main repo a while back about the use of print when referring to the console.

For consistency and accuracy, I vote we use either use `log a message to the console` or `output a message to the console` like MDN says.

https://developer.mozilla.org/en-US/docs/Web/API/console/log_static

Reply:
done!